schemas/contacts.js

```diff
@@ -0,0 +1,11 @@
+const Joi = require("joi");
+
+const addSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
add message to required

Reply:
Hi.
I checked "Joi" documentation, there is no information about message field in ".required( )". 
By "schema" testing it is working, I have message "{"message": "\"name\" is required"}" when I imitate put request without "name".
Can you please write where exactly I should add message?

Reply:
name: Joi.string()
    .messages({
      'string.empty': `"name" cannot be an empty field`,
    })
    
    you could do something like that

Reply:
> name: Joi.string() .messages({ 'string.empty': `"name" cannot be an empty field`, })
> 
> ```
> you could do something like that
> ```

Changed and tested, everything works.
"const addSchema = Joi.object({
  name: Joi.string()
    .required()
    .messages({ "any.required": 'missing required name field' }),
  email: Joi.string()
    .required()
    .messages({ "any.required": 'missing required email field' }),
  phone: Joi.string()
    .required()
    .messages({ "any.required": 'missing required phone field' }),
});"